README.md

```diff
@@ -39,6 +39,15 @@ Pkg.test("ArrayFire")
 ```
 If you have any issues getting `ArrayFire.jl` to work, please check the Troubleshooting section below. If it still doesn't work, please file an issue. 
 
+#Windows
+Just downdload the [installer](http://arrayfire.com/download/) (you need to create an account).
```

Comment:
Just fix the typo: download. 

Change line to: Download the official installer from [here](http://arrayfire.com/download/), after creating the account.